sirepo/job_supervisor.py

```diff
@@ -494,6 +495,18 @@ def _ops_to_cancel():
                 c.destroy(cancel=False)
 
     async def _receive_api_runSimulation(self, req):
+        def _set_error(op, compute_job_serial):
+            if self.db.computeJobSerial != compute_job_serial:
+                # We no longer own the db so there is nothing to do
+                return
+            self.__db_update(
+                status=job.ERROR,
+                error='Server error',
```

Comment:
I'm not sure. Are you imagining adding a field like "internalErrorMessage"? I think that would be useful because when a users sim goes wrong we can use the internalErrorMessage value as a jumping off point in the server logs. I don't know if displaying to the user the error is valuable though. Thoughts?